airbyte-integrations/connectors/source-aha/integration_tests/configured_catalog.json

```diff
@@ -1,14 +1,5 @@
 {
   "streams": [
-    {
```

Comment:
The configured catalog is used to run tests. If your account doesn't have permissions just remove it, but don't remove it Airbyte will test them.

Reply:
OK understood